mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -92,7 +92,7 @@ const ABCI_REQUEST_PROVE: bool = false;
 /// 0.25 is good average gas price on atom and iris
 const DEFAULT_GAS_PRICE: f64 = 0.25;
 pub(super) const TIMEOUT_HEIGHT_DELTA: u64 = 100;
-pub const GAS_LIMIT_DEFAULT: u64 = 100_000;
+pub const GAS_LIMIT_DEFAULT: u64 = 125_000;
```

Comment:
https://nyancat.iobscan.io/#/txs/ADA2A1B69E98A3E0BBF2A7183D8DC81C3F194E972A0C13312369B01039F168F3

Reply:
Could you document(either here in the PR comments or in the changelog) why you are changing this value?

Reply:
Because HTLCs requiring more gas than our limit lately. Since gas prices can be updated from the coins configuration, this should never be problem even if we make it `200_000`.